src/firefly/js/ui/FitsDownloadDialog.jsx

```diff
@@ -319,7 +283,7 @@ function makeFileName(plot, band='NO_BAND', ext= 'fits') {
             case Band.BLUE.key: root= 'image-blue'; break;
         }
     }
-    const title= req.getDownloadFileNameRoot() ? req.getDownloadFileNameRoot() : plot.title;
+    const title= req?.getDownloadFileNameRoot() ? req?.getDownloadFileNameRoot() : plot.title;
```

Comment:
This can be shorten ..
```
    const title= req?.getDownloadFileNameRoot() ?? plot.title;     
```
Or || instead of ?? depending on whether or not you allow blank.